WcaOnRails/app/models/concerns/localized_sortable.rb

```diff
@@ -38,7 +33,11 @@ def real
     end
 
     def all_sorted_by(locale, real: false)
-      real ? self::ALL_SORTED_BY_LOCALE[locale].select(&:real?) : self::ALL_SORTED_BY_LOCALE[locale]
+      @all_sorted_by_locale ||= Hash[I18n.available_locales.map do |available_locale|
```

Comment:
Using `@all` instead of `@@all` felt weird to me at first, but apparently classes in Ruby are instance themselves so it works...

Reply:
This stuff is all kind of black magic to me, but this change does seem to work!